### PR TITLE
Batch update deployment signalled tokens to Subgraph entity

### PR DESCRIPTION
**Pull request by juanmardefago**
Currently syncing here: https://thegraph.com/legacy-explorer/subgraph/juanmardefago/dev-subgraph?version=pending

Also fixed some inconsistent spacing on some files, and a weird conflict on a file that we stopped versioning a while ago.

### CODE REVIEW

File: src/mappings/gns.ts
Comment (nenadjaja) [inline src/mappings/gns.ts:173]: Should `let oldVersionID = subgraph.currentVersion` line of code be after this line? (After we set `subgraph.currentVersion`) ?

**Review by nenadjaja — COMMENTED** [review]
Had one question. 
Might want to wait for @davekaj for approval? Unless this is urgent

File: src/mappings/gns.ts
Comment (juanmardefago) [reply src/mappings/gns.ts:173]: We actually want to maintain the previous version id (before we set the new one, even if it's null) that's why I do it before ;)

File: src/mappings/metadataHelpers.ts
Comment (davekaj) [inline src/mappings/metadataHelpers.ts:7]: Hmm, i remember I did a commit where i made this file git ignored. in fact, `.gitignore` contains it right now. 

i think we can just delete this file from the PR

File: src/mappings/helpers.ts
Comment (davekaj) [inline src/mappings/helpers.ts:840]: What is the reason for including this field on the subgraph? Just curious

File: src/mappings/gns.ts
Comment (davekaj) [inline src/mappings/gns.ts:203]: I think this function, or one similar to it, will have to run on `handleSubgraphDeprecated`. it might be that we just need to lower the active count, and set active to false. wdyt?

**Review by davekaj — CHANGES_REQUESTED** [review]
A few questions from me

File: src/mappings/metadataHelpers.ts
Comment (juanmardefago) [reply src/mappings/metadataHelpers.ts:7]: Yeah, I'm not 100% sure why it was still being tracked, but since the old version contained conflict strings, I made sure to at least remove that.
I'm gonna go ahead and delete them on the update.

File: src/mappings/helpers.ts
Comment (juanmardefago) [reply src/mappings/helpers.ts:840]: So we can update the old deployment active subgraph count when it changes to a new deployment. (that counter basically adds a little bit of extra information so you can know how many subgraphs are actively using that deployment as their current version)

File: src/mappings/metadataHelpers.ts
Comment (davekaj) [reply src/mappings/metadataHelpers.ts:7]: yeah i think gitignore does a weird thing where if the file EXISTS it doesnt catch it, and will continue to allow you to edit it. but then once you delete it, gitignore works properly.

File: src/mappings/gns.ts
Comment (juanmardefago) [reply src/mappings/gns.ts:203]: 🤔 
maybe, yeah, although technically the subgraph still has the deployment as its' current version, so it might be weird when querying that data 🤔

Comment (juanmardefago) [reply src/mappings/gns.ts:203]: Maybe it would be best to track that as a separate count, like having `subgraphCount`, `activeSubgraphCount` and `deprecatedSubgraphCount`, since an active subgraph for the deployment can stop being active because it was upgraded or because it was deprecated

**Comment by juanmardefago** [comment]
@davekaj rebased to fix conflicts, and changed it to implement a different counter when deprecating, while reusing most of the original code.

**Comment by juanmardefago** [comment]
Currently syncing here: https://thegraph.com/legacy-explorer/subgraph/juanmardefago/dev-subgraph?version=pending

**Review by davekaj — APPROVED** [review]
lgtm

**Pull request merged**
